### PR TITLE
CLDR-13488 Make final changes in data, to change colon to AND

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13488
- [ ] Updated PR title and link in previous line to include Issue number

Made changed : to -and- to resolve issue in spec working doc (see Open Issues there).
Will use foot-and-inch in place of either foot:inch or foot+inch

### CODE REVIEW

File: tools/java/org/unicode/cldr/util/UnitPreferences.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public String toString() {
     }
 
     static final Splitter SPLIT_SPACE = Splitter.on(' ').trimResults().omitEmptyStrings();
-    static final Joiner JOIN_COLON = Joiner.on(':');
+    static final Joiner JOIN_AND = Joiner.on("-and");
```
Comment (sffc) [inline tools/java/org/unicode/cldr/util/UnitPreferences.java]: Missing a `-` ?

Comment (macchiati) [reply tools/java/org/unicode/cldr/util/UnitPreferences.java]: Good catch; the tests are not updated to check for that.

Comment (macchiati) [reply tools/java/org/unicode/cldr/util/UnitPreferences.java]: Will fix.

Comment (macchiati) [reply tools/java/org/unicode/cldr/util/UnitPreferences.java]: Fixed it; turns out that it was dead code; found another piece of dead code also.

**Pull request merged**